### PR TITLE
Update log format

### PR DESCRIPTION
**Pull request by LorcanMcVeigh**
### Proposed changes
update log format from 
`2020/04/10 21:13:43 Updated HTTP servers of {backend2 80 asg-sync-4-WebserverGroup2-B07B6PPXO49Y http 0xc0000a50e0 0xc0000a50e8 10s 0s}; Added: [{ID:0 Server:172.31.26.57:80 MaxConns:0xc0000a50e0 MaxFails:0xc0000a50e8 FailTimeout:10s SlowStart:0s Route: Backup:<nil> Down:<nil> Drain:false Weight:<nil> Service:}], Removed: [], Updated: []`
To new format 
`2020/04/10 21:13:43 Updated HTTP servers of backend2 for group asg-sync-4-WebserverGroup2-B07B6PPXO49Y: Added: [172.31.26.57:80,172.31.26.61:80], Removed: [], Updated: []`

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-asg-sync/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by Dean-Coakley — CHANGES_REQUESTED** [review]
Thanks for including a sample output in the PR description. - That's helpful!

File: cmd/sync/main.go [hunk 1]
```diff
@@ -164,3 +164,19 @@ func main() {
 		}
 	}
 }
+
+func getUpstreamServerAddress(server []nginx.UpstreamServer) []string {
```
[the remark this answers is not in the document]
Comment (Dean-Coakley) [reply cmd/sync/main.go:168]: @lucacome Should this func be renamed to `getUpstreamServerEndpoints` or am I spending too much time with Kubernetes? 😅

File: cmd/sync/main.go [hunk 2]
```diff
@@ -164,3 +164,19 @@ func main() {
...
+	}
+	return upstreamServerAddr
+}
+
+func getStreamUpstreamServerAddress(server []nginx.StreamUpstreamServer) []string {
+	var StreamUpstreamServerAddr []string
+	for _, s := range server {
+		StreamUpstreamServerAddr = append(StreamUpstreamServerAddr, s.Server)
+	}
+	return StreamUpstreamServerAddr
```
[the remark this answers is not in the document]
Comment (LorcanMcVeigh) [reply cmd/sync/main.go:181]: yes

File: cmd/sync/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (lucacome) [reply cmd/sync/main.go:172]: I think `Addresses` is good 😄

**Review by lucacome — APPROVED** [review]

File: cmd/sync/main.go [hunk 3]
```diff
@@ -164,3 +168,19 @@ func main() {
 		}
 	}
 }
+
+func getUpstreamServerAddress(server []nginx.UpstreamServer) []string {
```
Comment (Dean-Coakley) [inline cmd/sync/main.go:172]:
```suggestion
func getUpstreamServerAddresses(server []nginx.UpstreamServer) []string {
```

File: cmd/sync/main.go [hunk 4]
```diff
@@ -150,7 +154,7 @@ func main() {
 				}
 
 				if len(added) > 0 || len(removed) > 0 || len(updated) > 0 {
-					log.Printf("Updated Stream servers of %v; Added: %+v, Removed: %+v, Updated: %+v", upstream, added, removed, updated)
+					log.Printf("Updated Stream servers of %v for group %v ; Added: %+v, Removed: %+v, Updated: %+v", upstream.Name, upstream.ScalingGroup, getStreamUpstreamServerAddress(added), getStreamUpstreamServerAddress(removed), getStreamUpstreamServerAddress(updated))
```
Comment (Dean-Coakley) [inline cmd/sync/main.go:157]: I recommend making the same change as with HTTP to reduce line length.

(something like:)
```suggestion
					addedAddresses := getStreamUpstreamServerAddress(added)
					removedAddresses := getStreamUpstreamServerAddress(removed)
					updatedAddresses := getStreamUpstreamServerAddress(updated)
					log.Printf("Updated Stream servers of %v for group %v ; Added: %+v, Removed: %+v, Updated: %+v",
						upstream.Name, upstream.ScalingGroup, addedAddresses, removedAddresses, updatedAddresses
```

**Review by Dean-Coakley — APPROVED** [review]

**Review by Dean-Coakley — APPROVED** [review]

**Pull request merged**